components/ws-manager-bridge/src/prebuild-updater.ts

```diff
@@ -4,22 +4,126 @@
  * See License.AGPL.txt in the project root for license information.
  */
 
+import { inject, injectable } from "inversify";
 import { TraceContext } from "@gitpod/gitpod-protocol/lib/util/tracing";
-import { WorkspaceStatus } from "@gitpod/ws-manager/lib";
+import { WorkspaceStatus, WorkspaceType } from "@gitpod/ws-manager/lib";
 import { WorkspaceInstance } from "@gitpod/gitpod-protocol";
-import { injectable } from "inversify";
+import { log, LogContext } from "@gitpod/gitpod-protocol/lib/util/logging";
+import { PrebuildStateMapper } from "./prebuild-state-mapper";
+import { DBWithTracing, TracedWorkspaceDB } from "@gitpod/gitpod-db/lib/traced-db";
+import { WorkspaceDB } from "@gitpod/gitpod-db/lib/workspace-db";
+import { MessageBusIntegration } from "./messagebus-integration";
+import { PrometheusMetricsExporter } from "./prometheus-metrics-exporter";
+import { filterStatus } from "./bridge";
 
-export const PrebuildUpdater = Symbol("PrebuildUpdater");
+@injectable()
+export class PrebuildUpdater {
+    @inject(PrebuildStateMapper)
+    protected readonly prebuildStateMapper: PrebuildStateMapper;
 
-export interface PrebuildUpdater {
```

Comment:
diff is now weird that I renamed the original file, but effectively the only implementation of PrebuildUpdater is the one we had, and there's no interface anymore